scalardl-test/verification-config.toml

```diff
@@ -16,6 +16,8 @@
   concurrency = 5
   run_for_sec = 900
   ramp_for_sec = 0
+  sleep_base_millis = 1000
```

Comment:
I don't think they should be in common. I think they should be in the `test_config` since it's specific for `TransferChecke`, right?

Also, the variable names are too general. Please make them more specific.

Reply:
Thank you for the suggestion.
I will move those to `test_config` and rename those to be more specific.

Reply:
I have changed variable name and moved the parameters to `test_config`.